### PR TITLE
feat: add layout on module

### PR DESCRIPTION
**Pull request by ricardogobbosouza**
Improvement to add layout by module without the need to tweak the options.

**module.js**
```js
module.exports = function () {
  // Register `layout`
  this.addLayout(path.resolve(__dirname, 'layout.vue'))
  
  // Register `layout` with a custom name
  this.addLayout(path.resolve(__dirname, 'layout.vue'), 'custom-name')
}
```

### CODE REVIEW

File: lib/core/module.js [hunk 1]
```diff
@@ -71,6 +71,14 @@ module.exports = class ModuleContainer {
     })
   }
 
+  addLayout(template, name) {
+    const { dst, src } = this.addTemplate(template)
+    const srcPath = path.parse(src)
+
+    // Add to nuxt layouys
```
Comment (yashha) [inline lib/core/module.js]: typo

**Comment by Atinux** [comment]
Hi @ricardogobbosouza 

Thank you for this great PR, I really like this idea to give a shortcut for modules to register new layouts :)

**Comment by ricardogobbosouza** [comment]
Hi @Atinux,
Thanks, the idea is to simplify the use of the developer.

I'm thinking of implementing `addLayouts` that will go through a directory and create all the layouts inside it with namespace, what do you think?

**Comment by Atinux** [comment]
Actually, `addLayout` is enough :)

Sorry for the laaaate answer!

**Pull request merged**